Misc/NEWS.d/next/Library/2019-01-15-17-45-45.bpo-35701.KFYngZ.rst

```diff
@@ -0,0 +1 @@
+Re-enable the ability to weakly reference a UUID object.
```

Comment:
It seems like https://bugs.python.org/issue30977 was never released, the change was only made in the master branch. So no user got the issue with a public release. If I'm right, the NEWS entry is useless.

Reply:
I believe you are correct, I was a bit unsure if I needed to then mention it. Should I drop the commit and force push? It seems like the bot detects when there is no news blurb and throws an error, is this optional then?

Reply:
Remove the NEWS entry. I will add the "skip news" label, don't worry.